uwblueprint/starter-code-v2
# Add example context components for frontend state management

## Notion ticket link

[Setup Frontend State Management](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d?p=e57442ad5fd04ac3b5984a9b7318dde1)



## Implementation description
* Add example of how to use React Contexts to replace redux
* Add sample context representing team information
* Provides context to entire application
* Shows example of creating a dispatcher, action types, and a dispatch context



## Steps to test
1. Verify team info displays on default page.
2. Go to Edit Team Info page, play around with form settings and verify changes on home page. (Don't reload the app as this is all local app state and we are not wrtiting to any DB).



## What should reviewers focus on?
* `contexts/SampleContext.ts`
* `reducers/useSampleContextReducer.ts`
* `DefaultPage.tsx`
* `EditTeamInfoPage.tsx`

## Checklist
